docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,282 @@ Config option priority is applied in the following order:
+                "sqs:ReceiveMessage",
+                "sqs:DeleteMessage"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:sqs:eu-central-1:1234567890:audit-sns",
+            "Sid": "AllowReceiveSQS"
+        },
+        {
+            "Action": [
+                "glue:GetTable",
+                "athena:StartQueryExecution",
+                "athena:GetQueryResults",
+                "athena:GetQueryExecution"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:glue:eu-central-1:1234567890:table/audit_db/audit_table",
+                "arn:aws:glue:eu-central-1:1234567890:database/audit_db",
+                "arn:aws:glue:eu-central-1:1234567890:catalog",
+                "arn:aws:athena:eu-central-1:1234567890:workgroup/audit_workgroup"
+            ],
+            "Sid": "AllowAthenaQuery"
+        },
+        {
+            "Action": [
+                "kms:GenerateDataKey",
+                "kms:Decrypt"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:kms:eu-central-1:1234567890:key/kms_id",
+            "Sid": "AllowAthenaKMSUsage"
+        }
+    ]
+}
+```
+
+### Migration from Dynamo to Athena audit logs backend
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  Migration is only needed if Dynamo audit logs was used and you want to keep
+  old data.
+</Admonition>
+
+Migration consist of following steps:
+
+1. Set up Athena infrastructure
+1. Dual write to both Dynamo and Athena, query from Dynamo
+1. Migrate old data from Dynamo to Athena
+1. Dual write to both Dynamo and Athena, query from Athena
+1. Disable writing to Dynamo
+
+In teleport storage config, `audit_events_uri` accepts multiple URLs used as
+audit loggers. If more than 1 is used, then events are written to each audit
+system, and queries are executed from first one.
+
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  If anything goes wrong during migration steps 1-4, rollback to Dynamo solution
+  is as trivial as making sure it's on first place inside `audit_events_uri`
+  and removing Athena URL from `audit_events_uri`.
+</Admonition>
+
+#### Dual write to both Dynamo and Athena, query from Dynamo
+
+Second step of migration requires setting following configuration:
+
+```yaml
+teleport:
+  storage:
+    audit_events_uri: 
+    - 'dynamodb://events_table_name'
+    - 'athena://db.table?otherQueryParams'
+```
+
+When auth server is restarted, you should verify that parquet files are stored
+on S3 bucket specified using `locationS3` parameter.
+
+#### Migrate old data from Dynamo to Athena
+
+This step requires using client machine to export data from Dynamo and publish
+it to Athena logger. We recommend using for example EC2 instance with disk size
+at least 2x bigger than table size in dynamo.
+
+Instructions how to use migration tool can be found
+[here](https://github.com/gravitational/teleport/blob/master/examples/dynamoathenamigration/README.md).
```

Comment:
```suggestion
[on GitHub](https://github.com/gravitational/teleport/blob/master/examples/dynamoathenamigration/README.md).
```
It's usually poor practice to use "here" as link text since it obscures the target of the link.